sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/ExpressionConverter.java

```diff
@@ -551,6 +561,60 @@ public RexNode convertResolvedLiteral(ResolvedLiteral resolvedLiteral) {
     return ret;
   }
 
+  public RexCall convertTableValuedFunction(
```

Comment:
Please add a method comment since it is public and not 'simple' like a getter or setter.

Reply:
Done.